packages/transactions/src/cl.ts

```diff
@@ -18,6 +18,8 @@ import {
   uintCV,
 } from './clarity';
 
+import { prettyPrint } from './clarity/prettyPrint';
+
```

Comment:
I think this could even be an `export from`

Reply:
Will try, although it's rendered a bit differently in the docs

Reply:
addressed